src/js/utils/element-utils.js

```diff
@@ -0,0 +1,12 @@
+export const updateIconText = (svg, value, selector = '#value') => {
```

Comment:
this id `#value` should be more unique in the case we provide custom icons in the light DOM then there are 2 elements with the same id in the document. would using a class selector or even a tag name (`text`) selector be enough here?

Reply:
Good callout. I _*thought*_ internal SVG nodes' ids only had to be unique internal to the SVG, but I'm not 100% confident on that, so might as well change to a class selector just in case (intentionally wrote it as a selector to allow for the flexibility.